src/main/kotlin/no/nav/syfo/rules/ValidationRuleChain.kt

```diff
@@ -67,7 +67,9 @@ enum class ValidationRuleChain(
             infotrygdForesp.sMhistorikk?.sykmelding != null && infotrygdForesp.sMhistorikk.sykmelding
                     .any { infotrygdForespSykmelding ->
                         infotrygdForespSykmelding.periode?.arbufoerFOM != null && (sykmeldingPeriods.fom.isBefore(infotrygdForespSykmelding.periode.arbufoerFOM) ||
-                                sykmeldingPeriods.fom.isEqual(infotrygdForespSykmelding.periode.arbufoerFOM))
+                                sykmeldingPeriods.fom.isEqual(infotrygdForespSykmelding.periode.arbufoerFOM)) &&
```

Comment:
Jeg vet ikke om det er at det er sent på fredag eller ikke. Men jeg klarer ikke tolke disse reglene. Kan vi gjøre noe for å forenkle dem litt. For eksempel gjøre nullsjekker tidligere osv.

Reply:
Kan nok, det men tenker det er bedre å opprette ein rein pr, det vi ikkje sender inn hele infotrygdForesp, men lager vår eget data class som er letter å håndtere.